code/src/Components/List.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { useSelector} from 'react-redux';
+
+import { Item } from './Item';
+import { ListWrapper } from '../library/Containers';
+
+export const List = () => {
+  //This useSelector helps us get a hold of the items inside our store (tasks)
+
+  const items = useSelector((store) => store.tasks.items);
+
+  return (
+    <>
```

Comment:
You don't need this fraction since the `ListWrapper` is wrapping the return content.